geometry_msgs/Pose2D pose
float64 width
